### PR TITLE
Update release notes about basic_auth

### PR DESCRIPTION
**Pull request by brhou**
Basic auth is no longer a thing after nov 4th. Release noting about it.

### CODE REVIEW

File: source/release_notes.html [hunk 1]
```diff
@@ -345,6 +345,16 @@ <h1 class="title">Release Notes</h1>
 </ul>
 </td>
 </tr>
+<tr>
+<a name='november-4-2015'></a>
+November 4, 2015
+</td>
+<td>
+<ul>
+<li>Basic auth has been disabled for all accounts. Please contact support for any issues.</li>
```
Comment (steverice) [inline source/release_notes.html]: I think even this suggests that basic auth is something we can enable or disable, rather that it simply being gone entirely.
Our earlier message about this reads
> Support for HTTP Basic Authentication will no longer be provided.

Can we just say that the PagerDuty API no longer works with basic authentication?

File: source/release_notes.html [hunk 2]
```diff
@@ -345,6 +345,16 @@ <h1 class="title">Release Notes</h1>
 </ul>
 </td>
 </tr>
+<tr>
+<a name='november-4-2015'></a>
```
Comment (divtxt) [inline source/release_notes.html]: Missing `<td class='date'>`

**Comment by divtxt** [comment]
Newest entry should go to the top. (Looks like the previous one is at the bottom too)

File: source/release_notes.html (hunk 2 above)
Comment (brhou) [inline source/release_notes.html]: Good catch.

**Comment by steverice** [comment]
:+1:

**Comment by billkauf** [comment]
:+1:

**Comment by afolson** [comment]
Just commenting to apologize for how awful this process is. Kbye.

**Pull request merged**